scripts/single-node-upgrades.sh

```diff
@@ -100,9 +100,7 @@ startCelestiaApp() {
     --grpc.enable \
     --grpc-web.enable \
     --timeout-commit 1s \
-    --force-no-bbr \
-    --address tcp://127.0.0.1:26658 \
-    --proxy_app tcp://127.0.0.1:26658
```

Comment:
[question] should we hold off on merging this until:
1. https://github.com/celestiaorg/celestia-app/pull/4754 merges
2. Cut v3.x release
3. Bump in `Makefile`

Reply:
yes! let's do that

Reply:
Sounds good. I created https://github.com/celestiaorg/celestia-app/pull/4762